plugins/inputs/sql/README.md

```diff
@@ -0,0 +1,192 @@
+Supported/integrated drivers are mssql (SQLServer), mysql (MySQL), postgres (Postgres)
+Activable drivers (read below) are all golang SQL compliant drivers (see https://github.com/golang/go/wiki/SQLDrivers): for instance oci8 for Oracle or sqlite3 (SQLite)
+
+## Getting started :
+First you need to grant read/select privileges on queried tables to the database user you use for the connection
+
+### Non pure go drivers
+For some not pure go drivers you may need external shared libraries and environment variables: look at sql driver implementation site
+Actually the dependencies to all those drivers (oracle,db2,sap) are commented in the sql.go source. You can enable it, just remove the comment and perform a 'go get <driver git url>' and recompile telegraf. As alternative you can use the 'golang 1.8 plugins feature'like described here below
+
+### Oracle driver with golang 1.8 plugins feature
+Follow the docu in https://github.com/mattn/go-oci8 for build the oci8 driver.
+If all is going well now golang oci8 driver is compiled and linked against oracle shared libs. But not linked in telegraf.
+
+For let i use in telegraf, do the following:
+create a file plugin.go with this content:
+
+```
+package main
+
+import "C"
+
+import (
+	"log"
+	// .. here you can add import to other drivers
+	_ "github.com/mattn/go-oci8" // requires external prorietary libs
+	// _ "bitbucket.org/phiggins/db2cli" // requires external prorietary libs
+	// _ "github.com/mattn/go-sqlite3" // not compiles on windows
+)
+func main() {
+	log.Printf("I! Loaded plugin of shared libs")
+}
+``` 
+build it with
+``` 
+mkdir $GOPATH/lib
+go build -buildmode=plugin -o $GOPATH/lib/oci8_go.so plugin.go
+```
+in the input plugin configuration specigy the path of the created shared lib 
+``` 
+[[inputs.sql]]
+	...
+	driver = "oci8"
+	shared_lib = "/home/luca/.gocode/lib/oci8_go.so"
+	...
+``` 
+
+The steps of above can be reused in a similar way for other proprietary and non proprietary drivers
+
+
+## Configuration:
+
+```
+	[[inputs.sql]]
+		# debug=false						# Enables very verbose output
+
+		## Database Driver
+		driver = "mysql" 					# required. Valid options: mssql (SQLServer), mysql (MySQL), postgres (Postgres), sqlite3 (SQLite), [oci8 ora.v4 (Oracle)]
+		# shared_lib = "/home/luca/.gocode/lib/oci8_go.so"		# optional: path to the golang 1.8 plugin shared lib
+		# keep_connection = false 			# true: keeps the connection with database instead to reconnect at each poll and uses prepared statements (false: reconnection at each poll, no prepared statements)
+
+		## Server DSNs
+		servers  = ["readuser:sEcReT@tcp(neteye.wp.lan:3307)/rue", "readuser:sEcReT@tcp(hostmysql.wp.lan:3307)/monitoring"] # required. Connection DSN to pass to the DB driver
+		hosts=["neteye", "hostmysql"]	# optional: for each server a relative host entry should be specified and will be added as host tag
```

Comment:
ok:
I renamed _servers_ option into _db_source_names_ and _hosts_ into _servers_